projects/tuist/fixtures/ios_app_with_templates/Tuist/Templates/custom_using_copy_folder/custom_using_copy_folder.swift

```diff
@@ -0,0 +1,21 @@
+import ProjectDescription
+
+let nameAttributeFour: Template.Attribute = .required("name")
+let platformAttributeFour: Template.Attribute = .optional("platform", default: "ios")
+
+let testContentsFour = """
+// this is test \(nameAttributeFour) content
+"""
+
+let templateFour = Template(
+    description: "Custom template",
+    attributes: [
+        nameAttributeFour,
+        platformAttributeFour
+    ],
+    files: [
+        .string(path: "\(nameAttributeFour)/custom.swift", contents: testContentsFour),
+        .file(path: "\(nameAttributeFour)/generated.swift", templatePath: "platform_four.stencil"),
+        .directory(path: "\(nameAttributeFour)",sourcePath: "sourceFolder")
```

Comment:
thanks for doing this but I think it's better if this was part of [this](https://github.com/tuist/tuist/tree/main/projects/tuist/fixtures/ios_app_with_templates) already existing fixture rather than creating a new one that is more-or-less the same.

Reply:
never mind, I have not noticed it is a part of `app_with_templates`